my-app/src/App.css

```diff
@@ -0,0 +1,246 @@
+  background-position: right bottom;
+  min-height: 30vh;
+  max-height: 20vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  font-size: calc(10px + 2vmin);
+  color: white;
+  font-family: Avenir;
+  font-weight: normal;
+  text-shadow: 3px 2px rgb(49, 41, 41);
+}
+
+.App-link {
+  color: #f8f8f8;
+}
+
+@keyframes App-logo-spin {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
+
+/* @media (prefers-reduced-motion: no-preference) {
+  button[class^='btn-']:hover{
+    animation: App-logo-spin infinite 3s linear;
+  }
+}
+@keyframes App-logo-spin {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+} */
+/**/
+
+.box {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  align-content: stretch;
+  position: relative;
+  margin: auto;
+  margin-top: 5%;
+  min-height: 50vh;
+}
+
+.principal {
+  align-items: center;
+  display: flex;
+  height: 100vh;
+}
+
+.secundario {
+  margin: 0 auto;
+}
+.overlay {
+  background: rgba(0, 0, 0, 0.4);
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  align-items: center;
+  justify-content: center;
+  display: flex;
+  visibility: hidden;
+}
+
+.overlay.active {
+  visibility: visible;
+}
+.popup.popup {
+  background: #f8f8f8;
+  box-shadow: 0px 0px 5px 0px rgba(0, 0, 0, 0.3);
+  border-radius: 3px;
+  font-family: 'Montserrat', sans-serif;
+  padding: 20px;
+  text-align: center;
+  width: 600px;
+
+  transition: 0.3s ease all;
+  transform: scale(0.7);
+  /* opacity: 0; */
+
+  border-radius: 8px;
+  color: black;
+  position: 'absolute';
+  top: 50%;
+  left: 50%;
+  transform: 'translate(-50%, -50%)';
+}
+
+.popup.card {
+  transform: scale(1);
+  background: #f8f8f8;
+  box-shadow: 0px 0px 5px 0px rgba(0, 0, 0, 0.3);
+  border-radius: 3px;
+  font-family: 'Montserrat', sans-serif;
+  padding: 20px;
+  text-align: center;
+  width: 600px;
+  transition: 0.3s ease all;
+  /*transform: scale(1.5);
+    /* opacity: 0; */
+  border-radius: 8px;
+  position: 'absolute';
+  top: 50%;
+  left: 50%;
+
+  max-height: 700px;
+  overflow: auto;
+}
+.popup.card.fotito {
+  min-width: 100%;
+}
+
+button[class^='btn-'] {
+  text-align: center;
+  background-color: rgba(103, 58, 183, 1);
+  /* rgb(192, 83, 92); */
+  opacity: 0.5;
+  color: white;
+  border-color: transparent;
+  width: 10em;
+  height: 2.3em;
+  border: 1px solid rgb(229, 229, 229);
+  border-radius: 1.88rem;
+}
+
+button[class^='btn-']:hover {
+  opacity: 1;
+}
+
+select,
+input {
+  /* display: flex; */
+  justify-content: center;
+  align-items: flex-start;
+  /*height: 150%;*/
+  background: transparent;
+  border: none;
+  text-decoration: underline;
+  /*font-size: 1.8em;*/
+  margin-left: auto;
+  text-decoration-thickness: initial;
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
+}
+
+.main-footer {
+  color: #f8f8f8;
+  background-color: black;
```

Comment:
It's a good practice to unify the way of using colors.

Reply:
Like if you use hex, use hex in all the app, don't mix it up with RGB or color names.